axum/src/docs/middleware.md

```diff
@@ -61,12 +61,12 @@ let app = Router::new()
 
 Some commonly used middleware are:
 
-- [`TraceLayer`](tower_http::trace) for high level tracing/logging.
-- [`CorsLayer`](tower_http::cors) for handling CORS.
-- [`CompressionLayer`](tower_http::compression) for automatic compression of
+- [`TraceLayer`](tower_http::trace::TraceLayer) for high level tracing/logging.
+- [`CorsLayer`](tower_http::cors::CorsLayer) for handling CORS.
+- [`CompressionLayer`](tower_http::compression::CompressionLayer) for automatic compression of
   responses.
-- [`RequestIdLayer`](tower_http::request_id) and
-  [`PropagateRequestIdLayer`](tower_http::request_id) set and propagate request
+- [`SetRequestIdLayer`](tower_http::request_id::SetRequestIdLayer) and
+  [`PropagateRequestIdLayer`](tower_http::request_id::PropagateRequestIdLayer) set and propagate request
   ids.
 - [`TimeoutLayer`](tower::timeout::TimeoutLayer) for timeouts. Note this
```

Comment:
Wanna change this to link to https://docs.rs/tower-http/latest/tower_http/timeout/index.html instead?

Reply:
So I should add another commonly used middleware, and it would look like this:
- [`TraceLayer`](docs,rs link) for high level tracing/logging.
- [`CorsLayer`](docs,rs link) for handling CORS.
- [`CompressionLayer`](docs,rs link) for automatic compression of responses.
- [`RequsetIdLayer`](docs,rs link) set and propagate requests IDs.
- [`tower::TimeoutLayer`](docs,rs link) and [`tower_http::TimeoutLayer`](docs,rs link) for timeouts.

Reply:
No need to mention `tower::TimeoutLayer`. The one in tower-http is what people should use.

And the links should to be to the modules, not the layers themselves. For example https://docs.rs/tower-http/latest/tower_http/trace/